### PR TITLE
This commit contains the updated app, model, controller and test file…

### PR DESCRIPTION
**Pull request by JamesLovesay**
Contains the app, controller, model and test files for the article/:article_id endpoint

### CODE REVIEW

File: __tests__/app.test.js [hunk 1]
```diff
@@ -35,4 +35,44 @@ describe('app', () => {
...
+                expect(article.body[0]).toEqual(
+                    expect.objectContaining({
+                    article_id: 1,
+                    title: "Living in the shadow of a great man",
+                    topic: "mitch",
+                    author: "butter_bridge",
+                    body: "I find this existence challenging",
+                    created_at: expect.any(String),
+                    votes: 100,
+                  }))
```
Comment (daviidbartlett) [inline __tests__/app.test.js:56]: don't expect this to be an array

File: controllers/controllers.js [hunk 2]
```diff
@@ -1,8 +1,23 @@
...
+        next(err)
+    })
 }
 
+exports.getArticle = (req, res, next) => {
+    const articleID  = req.params.article_id;
+    console.log(articleID)
+    selectArticle(articleID).then((article) => {
+        console.log('Hello', '<--------------------------')
+        res.status(200).send(article);
```
Comment (daviidbartlett) [inline controllers/controllers.js:17]: remove console.log

File: models/models.js [hunk 3]
```diff
@@ -4,4 +4,14 @@ exports.selectTopics = () => {
     return db.query('SELECT * FROM topics;').then(({rows}) => {
         return rows;
     })
+}
+
+exports.selectArticle = (id) => {
+    return db.query('SELECT * FROM articles WHERE article_id = $1', [id]).then(({ rows }) => {
+        console.log(rows.length, '<-------------------')
+        if (rows.length === 0) {
```
Comment (daviidbartlett) [inline models/models.js:12]: remove console.log

**Review by popatre — APPROVED** [review]
Changes have been corrected - free to merge.

**Pull request merged**